snap-build/config_amd64.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+local arch_qemu="x86_64"
```

Comment:
I don't think you can use `local` here as these are globals and `local` is for functions.

Reply:
you can if you source it inside a function  :smile:

Reply:
Ooooh - that's clever! I hadn't spotted that ;)